index.css

```diff
@@ -0,0 +1,231 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.d-grid {
+  display: grid;
+  grid-template-columns: auto;
+  grid-gap: 1rem;
+}
+
+.headericons {
+  width: 100%;
+}
+
+.margin {
+  margin: 30px;
+}
+
+.recent {
+  font-size: 30px;
+  font-family: "Crete Round", serif;
+  text-align: center;
+  align-items: center;
+}
+
+.container {
+  background-color: white;
+  padding: 1rem;
+}
+
+.line {
+  border-bottom: 3px solid #212121;
+  padding-top: 2rem;
+}
+
+.multi {
+  font-weight: 400;
+  font-family: "Crete Round", serif;
+}
+
+.acrobat {
+  width: 100%;
+}
+
+li {
+  list-style: none;
+}
+
+.daily {
+  text-align: left;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+}
+
+.btn {
+  margin: 3px;
+  background-color: white;
+  border: 1px solid rgb(113, 109, 109);
+  padding: 10px;
+}
+
+.btn-grid {
+  padding: 0;
+  display: flex;
+}
+
+.see-project {
+  background-color: #ff6b00;
+  width: 124px;
+  height: 48px;
+  border-radius: 0;
+  border-style: none;
+  color: white;
+  font-family: "Intter", serif;
+  font-size: 17px;
+  line-height: 24px;
+  align-items: center;
+  font-weight: 700;
+}
+
+a {
+  text-decoration: none;
+  font-family: "Inter", serif;
+  font-size: 17px;
+  color: white;
+}
+
+.last-part {
+  background-image: url(./images/secondpage/seeproject.svg);
+  object-fit: cover;
+  height: 350px;
+  text-align: left;
+  display: flex;
+  padding-top: 84px;
+  flex-direction: column;
+  justify-content: space-between;
+}
```

Comment:
It is very good you added the image as a background image
_I think it would be better if the images doesn't repeat itself when stretched._

Reply:
Well done @mrparkersson 💪🏻. This is a nice and polite comment review. However it does not point to any problem found in your code.

> A good comment includes a polite suggestion to an issue they found in code, it is then followed by an explanation why this change should be made.

Examples of issues you may look for when performing the review:

- Lack of semantic HTML tags use.
- Not applying HTML & CSS best practices correclty.
- Bugs missed the first time the project was built.